### PR TITLE
Delete all external axioms on base file

### LINKED ISSUE #2263: Uberon BASE is not a true BASE
The Uberon base file contains external axioms. Something is wrong with the makefile, my suspicion it it has the to with some import merging step somewhere.

Please fix this before the next release.

### PR DESCRIPTION
**Pull request by anitacaron**
Partially fixes #2263.

List of definitions that are different in UBERON and source ontology.

|Term ID|Def in UBERON|Def in source|
|---|---|---|
|RO:0002170|"Binary relationship: x connected_to y if and only if there exists some z such that z connects x and y in a ternary connected_to(x,y,z) relationship." [http://purl.obolibrary.org/obo/uberon/docs/Connectivity-Design-Pattern]|a is connected to b if and only if a and b are discrete structure, and there exists some connecting structure c, such that c connects a and b|
|RO:0002176|"Binary relationship: z connects x if and only if there exists some y such that z connects x and y in a ternary connected_to(x,y,z) relationship." [http://purl.obolibrary.org/obo/uberon/docs/Connectivity-Design-Pattern]|c connects a if and only if there exist some b such that a and b are similar parts of the same system, and c connects b, specifically, c connects a with b. When one structure connects two others it unites some aspect of the function or role they play within the system.|
|RO:0002256|"t1 developmentally_induced_by t2 if there is a process of organ induction (GO:0001759) with t1 and t2 as interacting participants. t2 causes t1 to change its fate from a precursor tissue type T to T', where T' develops_from T." [GO:0001759]|t1 induced_by t2 if there is a process of developmental induction (GO:0031128) with t1 and t2 as interacting participants. t2 causes t1 to change its fate from a precursor anatomical structure type T to T&apos;, where T&apos; develops_from T|
|RO:0002229|"Relation between occurrents, shares an end boundary with." [Allen:starts, https://orcid.org/0000-0002-6601-2165, ZFS:finishes]|inverse of ends with|
|RO:0002493|"Relation between continuant and occurrent, such that c ceases to exist at the end of p." [https://orcid.org/0000-0002-6601-2165]|x existence ends with y if and only if the time point at which x ends is equivalent to the time point at which y ends. Formally: x existence ends with y iff ω(x) = ω(y).|
|RO:0002488|"Relation between continuant c and occurrent s, such that every instance of c comes into existing during some s." [https://orcid.org/0000-0002-6601-2165]|x existence starts during y if and only if the time point at which x starts is after or equivalent to the time point at which y starts and before or equivalent to the time point at which y ends. Formally: x existence starts during y iff α(x) &gt;= α(y) &amp; α(x) &lt;= ω(y).|
|RO:0002568|"m1 has_muscle_antagonist m2 iff m1 acts in opposition to m2, and m2 is responsible for returning the structure to its initial position." [Wikipedia:Antagonist_(muscle)]|m1 has_muscle_antagonist m2 iff m1 has_muscle_insertion s, m2 has_muscle_insection s, m1 acts in opposition to m2, and m2 is responsible for returning the structure to its initial position.|
|RO:0002373|"m has_muscle_insertion s iff m is attaches_to s, and it is the case that when m contracts, s moves. Insertions are usually connections of muscle via tendon to bone." [Wikipedia:Insertion_(anatomy)]|m has_muscle_insertion s iff m is attaches_to s, and it is the case that when m contracts, s moves. Insertions are usually connections of muscle via tendon to bone.|
|RO:0002372|def: "m has_muscle_origin s iff m is attaches_to s, and it is the case that when m contracts, s does not move. The site of the origin tends to be more proximal and have greater mass than what the other end attaches to." [Wikipedia:Insertion_(anatomy)#Muscles]|m has_muscle_origin s iff m is attached_to s, and it is the case that when m contracts, s does not move. The site of the origin tends to be more proximal and have greater mass than what the other end attaches to.|
|RO:0002160|def: "S only_in_taxon T iff: S SubClassOf in_taxon only T." [PMID:20973947]|x only in taxon y if and only if x is in taxon y, and there is no other organism z such that y!=z a and x is in taxon z.|
|BSPO:0001108|"X distalmost_part_of Y <=> X is part_of Y and X is adjacent_to the distal boundary of Y." [https://orcid.org/0000-0002-6601-2165]|"X distalmost part of Y <=> X is in distal side of Y and X is adjacent to the distal boundary of Y" [https://orcid.org/0000-0002-6601-2165]|
|BSPO:0000098|"x dorsal_to y iff x is further along the dorso-ventral axis than y, towards the back. A dorso-ventral axis is an axis that bisects an organism from back (e.g. spinal column) to front (e.g. belly)." [http://orcid.org/0000-0002-6601-2165]|"x dorsal to y iff x is further along the dorso-ventral axis than y, towards the back. A dorso-ventral axis is an axis that extends through an organism from back (e.g. spinal column) to front (e.g. belly)." [BSPO:cjm]|
|BSPO:0001100|"X superficial_part_of Y <=> if Y is subdivided into two superficial and deep portions, X is part_of the superficial portion." [http://orcid.org/0000-0002-6601-2165]|"X superficial part of Y <=> if Y is subdivided into two superficial and deep portions, X is part of the superficial portion." [BSPO:cjm]|
|BSPO:0001107|"This relation holds when both the deep_to and ajdacent_to relationship similarly hold." [http://orcid.org/0000-0002-6601-2165]|"x immediately deep to y iff x deep_to y and x is contiguous with y." [BSPO:curators]|
|BSPO:0000099|"x posterior_to y iff x is further along the antero-posterior axis than y, towards the body/tail. An antero-posterior axis is an axis that bisects an organism from head end to opposite end of body or tail." [http://orcid.org/0000-0002-6601-2165]|"x posterior to y iff x is further along the antero-posterior axis than y, towards the body/tail. An antero-posterior axis is an axis that extends through an organism from head end to opposite end of body or tail." [BSPO:cjm]|
|BSPO:0000100|"x proximal_to y iff x is closer to the point of attachment with the body than y." [http://orcid.org/0000-0002-6601-2165]|"x proximal to y iff x is closer to the point of attachment with the body than y." [BSPO:cjm]|
|BSPO:0001106|"X proximalmost_part_of Y <=> X is part_of Y and X is adjacent_to the proximal boundary of Y." [https://orcid.org/0000-0002-6601-2165]|"X proximalmost part of Y <=> X is in proximal side of Y and X is adjacent to the proximal boundary of Y" [https://orcid.org/0000-0002-6601-2165]|
|BSPO:0000102|"x ventral_to y iff x is further along the dorso-ventral axis than y, towards the front. A dorso-ventral axis is an axis that bisects an organism from back (e.g. spinal column) to front (e.g. belly)." [http://orcid.org/0000-0002-6601-2165]|"x ventral to y iff x is further along the dorso-ventral axis than y, towards the front. A dorso-ventral axis is an axis that extends through an organism from back (e.g. spinal column) to front (e.g. belly)." [BSPO:cjm]|
|BSPO:0000096|"x anterior_to y iff x is further along the antero-posterior axis than y, towards the head. An antero-posterior axis is an axis that bisects an organism from head end to opposite end of body or tail: bearer." [http://orcid.org/0000-0002-6601-2165]|"x anterior to y iff x is further along the antero-posterior axis than y, towards the head. An antero-posterior axis is an axis that extends through an organism from head end to opposite end of body or tail." [BSPO:cjm]|

List of properties that all the axioms in UBERON don't exist in source. (No modifications were done, so cannot see in the diffs)

|Term ID|
|---|
|RO:0002082|
|RO:0002376|
|RO:0002215|
|RO:0002322|
|RO:0002170|
|RO:0002176|
|RO:0002150|
|RO:0002433|
|RO:0002256|
|RO:0002226|
|RO:0002569|
|BSPO:0000097|

### CODE REVIEW

**Comment by shawntanzk** [comment]
Avoid changing release artefacts goals.
Edit the edit file and remove the external axioms from the edit file.

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224025,8 +223998,6 @@ id: dorsal_to
 name: dorsal_to
 def: "x dorsal_to y iff x is further along the dorso-ventral axis than y, towards the back. A dorso-ventral axis is an axis that bisects an organism from back (e.g. spinal column) to front (e.g. belly)." [http://orcid.org/0000-0002-6601-2165]
```
Comment (matentzn) [inline src/ontology/uberon-edit.obo:223999]: delete def?

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -223988,7 +223962,6 @@ id: distalmost_part_of
 name: distalmost_part_of
 def: "X distalmost_part_of Y <=> X is part_of Y and X is adjacent_to the distal boundary of Y." [https://orcid.org/0000-0002-6601-2165]
```
Comment (matentzn) [inline src/ontology/uberon-edit.obo:223963]: delte def?

File: src/ontology/uberon-edit.obo [hunk 3]
```diff
@@ -223940,16 +223919,12 @@ xref: RO:0002285
 id: develops_from
 name: develops_from
 xref: RO:0002202
-is_transitive: true
 is_a: has_developmental_contribution_from ! has developmental contribution from
```
Comment (matentzn) [inline src/ontology/uberon-edit.obo:223922]: so this isa is not in the import?

File: src/ontology/uberon-edit.obo [hunk 4]
```diff
@@ -224059,14 +224028,11 @@ name: ends
 def: "Relation between occurrents, shares an end boundary with." [Allen:starts, https://orcid.org/0000-0002-6601-2165, ZFS:finishes]
```
Comment (matentzn) [inline src/ontology/uberon-edit.obo:224028]: delete rel?

File: src/ontology/uberon-edit.obo [hunk 5]
```diff
@@ -224080,41 +224046,29 @@ name: existence ends during
 def: "Relation between continuant c and occurrent s, such that every instance of c ceases to exist during some s, if it does not die prematurely." [https://orcid.org/0000-0002-6601-2165]
```
Comment (matentzn) [inline src/ontology/uberon-edit.obo:224046]: delete def?

Comment (matentzn) [reply src/ontology/uberon-edit.obo:224046]: or not in import?

File: src/ontology/uberon-edit.obo [hunk 6]
```diff
@@ -224080,41 +224046,29 @@ name: existence ends during
...
-holds_over_chain: developmentally_succeeded_by existence_ends_during_or_before
-holds_over_chain: part_of existence_ends_during_or_before
-transitive_over: part_of ! part of
-transitive_over: precedes ! precedes
-transitive_over: simultaneous_with ! simultaneous_with
 
 [Typedef]
 id: existence_ends_with
 name: existence ends with
 def: "Relation between continuant and occurrent, such that c ceases to exist at the end of p." [https://orcid.org/0000-0002-6601-2165]
```
Comment (matentzn) [inline src/ontology/uberon-edit.obo:224058]: Why not delete?

File: src/ontology/uberon-edit.obo [hunk 7]
```diff
@@ -224080,41 +224046,29 @@ name: existence ends during
...
 id: existence_starts_and_ends_during
 name: existence starts and ends during
 xref: RO:0002491
-is_a: existence_ends_during ! existence ends during
-is_a: existence_starts_during ! existence starts during
 
 [Typedef]
 id: existence_starts_during
 name: existence starts during
 def: "Relation between continuant c and occurrent s, such that every instance of c comes into existing during some s." [https://orcid.org/0000-0002-6601-2165]
```
Comment (matentzn) [inline src/ontology/uberon-edit.obo:224069]: Why not delete?

File: src/ontology/uberon-edit.obo (hunk 3 above)
Comment (anitacaron) [reply src/ontology/uberon-edit.obo:223922]: No, there's another one.

File: src/ontology/uberon-edit.obo (hunk 2 above)
Comment (anitacaron) [reply src/ontology/uberon-edit.obo:223963]: All the definitions I didn't delete are different in the import files. Maybe it's an updated version?

Comment (matentzn) [reply src/ontology/uberon-edit.obo:223963]: If there is a definition present in the import at all, delete this one here. If there is not, make an issue at repo of imported ontology!

File: src/ontology/uberon-edit.obo (hunk 3 above)
Comment (matentzn) [reply src/ontology/uberon-edit.obo:223922]: problematic.. logical axioms not present in the import need to be investigates. What is the parent in the other one?

Comment (anitacaron) [reply src/ontology/uberon-edit.obo:223922]: RO_0002258: developmentally preceded by

Comment (matentzn) [reply src/ontology/uberon-edit.obo:223922]: Alright, here is my suggestion: deal with the definitions for now, but leave the logical axioms that are _not base_. Then we deal with the logical axioms in a separate PR, checking diffs etc.

File: src/ontology/uberon-edit.obo (hunk 5 above)
Comment (anitacaron) [reply src/ontology/uberon-edit.obo:224046]: This definition seems to be an updated version.
In the `ro_import.owl` the definition is simple "inverse of ends with".
Maybe update definition in source?

File: src/ontology/uberon-edit.obo (hunk 7 above)
Comment (anitacaron) [reply src/ontology/uberon-edit.obo:224069]: The definitions are quite different. 
From `ro_import.owl` we have "x existence ends during y if and only if the time point at which x ends is before or equivalent to the time point at which y ends and after or equivalent to the point at which y starts. Formally: x existence ends during y iff ω(x) &lt;= ω(y) and ω(x) &gt;= α(y)."

File: src/ontology/uberon-edit.obo [hunk 8]
```diff
@@ -223695,17 +223690,14 @@ is_metadata_tag: true
 
 [Typedef]
 id: attaches_to
-name: attaches_to
+name: attached_to
```
Comment (matentzn) [inline src/ontology/uberon-edit.obo:223693]: Why the label change?

File: src/ontology/uberon-edit.obo [hunk 9]
```diff
@@ -223716,30 +223708,27 @@ xref: UBPROP:0000002
 is_metadata_tag: true
 
 [Typedef]
-id: bearer_of
-name: bearer of
+id: has_characteristic
```
Comment (matentzn) [inline src/ontology/uberon-edit.obo:223711]: Is this relationship used in Uberon? If not, remove it better.

File: src/ontology/uberon-edit.obo [hunk 10]
```diff
@@ -223716,30 +223708,27 @@ xref: UBPROP:0000002
...
+name: has characteristic
 xref: RO:0000053
 
 [Typedef]
-id: boundary_of
-name: boundary of
+id: 2D_boundary_of
```
Comment (matentzn) [inline src/ontology/uberon-edit.obo:223716]: Is this relationship used in uberon? if not, remove it..

File: src/ontology/uberon-edit.obo (hunk 6 above)
Comment (matentzn) [reply src/ontology/uberon-edit.obo:224058]: The definition below I mean.

File: src/ontology/uberon-edit.obo (hunk 7 above)
Comment (matentzn) [reply src/ontology/uberon-edit.obo:224069]: Ask in OBO uberon slack, stating that the same relation "x" has two definitions, and whether it is ok to delete the uberon one.

File: src/ontology/uberon-edit.obo [hunk 11]
```diff
@@ -224356,7 +224293,7 @@ is_metadata_tag: true
 
 [Typedef]
 id: homologous_to
-name: homologous_to
+name: shares ancestor with
```
Comment (matentzn) [inline src/ontology/uberon-edit.obo:224296]: suprising name change?

**Review by matentzn — COMMENTED** [review]
Excellent work @anitacaron very meticulous. Can you confirm once more that you have not removed any logical axioms that are _not_ in the imported files?

File: src/ontology/uberon-edit.obo (hunk 7 above)
Comment (anitacaron) [reply src/ontology/uberon-edit.obo:224069]: I'm listing all the different definitions, and I'll ask there.

File: src/ontology/uberon-edit.obo (hunk 9 above)
Comment (anitacaron) [reply src/ontology/uberon-edit.obo:223711]: Yes, it's being used in many terms.

File: src/ontology/uberon-edit.obo (hunk 8 above)
Comment (anitacaron) [reply src/ontology/uberon-edit.obo:223693]: Because the source is this one that I changed.

File: src/ontology/uberon-edit.obo (hunk 9 above)
Comment (matentzn) [reply src/ontology/uberon-edit.obo:223711]: but if you change the id, dont you have to change 1000 other things in the uberon-edit.obo file? Best not do all these changes in this PR: I would recommend leaving the `id` as is and only changing the `name:`? Or what do you think?

Comment (anitacaron) [reply src/ontology/uberon-edit.obo:223711]: Ah no, sorry, I was confused by another property that I only changed the name.

File: src/ontology/uberon-edit.obo (hunk 8 above)
Comment (anitacaron) [reply src/ontology/uberon-edit.obo:223693]: This one is being used in many terms. It's a symmetric property.

**Review by cmungall — APPROVED** [review]

**Comment by matentzn** [comment]
Well, if @cmungall approves, you can delete all of the definitions.. Maybe have @dosumis do one sweep as well.

**Comment by dosumis** [comment]
I'll have a quick check through of defs, but in general I'm happy to go with RO defs as these are the ones I've worked on.  @cmungall is probably be better placed to spot anything important that might be lost.

**Comment by dosumis** [comment]
Have now reviewed text defs.  Happy for all the Uberon defs listed in the table to be deleted.

In these 2 cases, we should probably add the more accessible def from Uberon as editor note or comment to RO.

RO:0002493 | "Relation between continuant and occurrent, such that c ceases to exist at the end of p." [https://orcid.org/0000-0002-6601-2165] | x existence ends with y if and only if the time point at which x ends is equivalent to the time point at which y ends. Formally: x existence ends with y iff ω(x) = ω(y).
-- | -- | --
RO:0002488 | "Relation between continuant c and occurrent s, such that every instance of c comes into existing during some s." [https://orcid.org/0000-0002-6601-2165] | x existence starts during y if and only if the time point at which x starts is after or equivalent to the time point at which y starts and before or equivalent to the time point at which y ends. Formally: x existence starts during y iff α(x) >= α(y) & α(x) <= ω(y).

**Comment by anitacaron** [comment]
Besides the definition, there isn't any difference for the same property. Instead, all logical axioms in UBERON don't exist in the source.

**Review by matentzn — APPROVED** [review]
Excellent work! I would like @shawntanzk when he is back to make sure the diff looks ok, then he can merge this. Super, super cool.

**Comment by shawntanzk** [comment]
not 100% sure what im suppose to look out for, but it doesn't seem to affect the class hierarchy of uberon (seeing the diff on simple file, the only changes seem to be what you changed, rather than axioms in uberon changing as a result) - which I assume is a good thing, and hence don't think there's an issue which this at all?

**Pull request merged**